presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -90,6 +92,29 @@ public ServerConfig setCatalogServerEnabled(boolean catalogServerEnabled)
         return this;
     }
 
+    public boolean isCoordinatorSidecar()
+    {
+        return coordinatorSidecar;
+    }
+
+    @Config("coordinator-sidecar")
+    public ServerConfig setCoordinatorSidecar(boolean coordinatorSidecar)
+    {
+        this.coordinatorSidecar = coordinatorSidecar;
+        return this;
+    }
```

Comment:
We don't need these because it will only be in C++

Reply:
I am actually using this here :

https://github.com/prestodb/presto/blob/874e168f4bcc0b4284811f1fd05df78fb8d868d1/presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java#L691

Reply:
It will only be true for C++ so let's just hard code that as false.